source/apps/export.txt

```diff
@@ -0,0 +1,210 @@
+.. _export-app:
+
+=============
+Export an App
+=============
+
+Overview
+--------
+
+You can download a directory of your App's configuration files by
+exporting them. This allows you to save your app's configuration in
+source control and to work locally with {+cli+} and your preferred
+text editor.
+
+The exported directory contains configuration files for most components
+of your App, including Data Sources, Functions, Triggers, and other
+services. For more information on the contents of an exported App, see
+:ref:`App Configuration <app-configuration>`.
+
+The export does not include any :ref:`Secrets <app-secret>`. If you
+create a new App based on exported configuration files, you must
+manually add Secrets to the new App. For more information, see
+:ref:`Copy an App <copy-app>`.
+
+.. include:: /includes/note-secrets-not-exported.rst
```

Comment:
Remove lines 20-23? They're redundant with this included note.

Reply:
I actually meant to delete the note!